systems/controllers/osc/operational_space_control.cc

```diff
@@ -381,8 +393,18 @@ VectorXd OperationalSpaceControl::SolveQp(
     const VectorXd& x_w_spr, const VectorXd& x_wo_spr,
     const drake::systems::Context<double>& context, double t, int fsm_state,
     double time_since_last_state_switch) const {
-  vector<bool> active_contact_flags = CalcActiveContactIndices(fsm_state);
+  // Get active contact indices
+  vector<int> active_contact_list = {};
+  if (single_contact_mode_) {
+    active_contact_list = contact_indices_map_.at(-1);
+  } else {
+    auto it_map = contact_indices_map_.find(fsm_state);
```

Comment:
we should either be doing something if the fsm_state isn't found, or just not checking for the fsm_state here and doing the check elsewhere.

Reply:
Currently, if the `fsm_state` is not found in the map, it uses an empty contact set. What do you think we should do in this case?

Reply:
An empty contact set should work, we should check to make sure the fsm_state is valid though. It might lead to hard to debug errors further later if we just assume there are no active contacts if we can't find the fsm_state.

Reply:
Ok, I added a little print message so that it's easy to spot the error. If we find a better solution later, we can update this